recipes/giflib/all/conanfile.py

```diff
@@ -102,7 +104,7 @@ def build_configure(self):
         env_build = AutoToolsBuildEnvironment(self, win_bash=self.settings.os == "Windows" and
                                               platform.system() == "Windows")
         if self.settings.os != "Windows":
-            env_build.fpic = self.options.fPIC
+            env_build.fpic = self.options.get_safe("fPIC", True)
```

Comment:
```suggestion
```
Not required actually. It's automatically configured: https://docs.conan.io/en/latest/reference/build_helpers/autotools.html#fpic

Reply:
if statement should be removed also.

Reply:
Yes.

Reply:
fixed